src/test/java/com/nextstep/racingcar/CarTest.java

```diff
@@ -22,39 +20,55 @@ void setUp() {
   }
 
   @Test
-  void createNewCars() {
-    List<Car> cars = new ArrayList<Car>();
-    String[] carNames = new String[]{"name", "asd"};
-    for (String carName : carNames) {
-      cars.add(new Car(carName));
-    }
+  void validCarName() {
+    assertThat(new Car("valid").getName()).isEqualTo("valid");
+  }
 
-    for (int i = 0; i < carNames.length; i++) {
-      assertThat(cars.get(i).getName())
-          .isEqualTo(carNames[i]);
-    }
+  @Test
+  void invalidCarName() {
+    assertThatExceptionOfType(RuntimeException.class).
```

Comment:
이왕 만들어둔 `InvalidNameFormatException`을 썼으면 더 좋았을텐데요
```suggestion
    assertThatExceptionOfType(InvalidNameFormatException.class).
```